.github/workflows/build_and_release.yml

```diff
@@ -90,7 +90,7 @@ jobs:
       - name: Update package name in pyproject.toml
         run: sed -i '/#replace_package_name_marker/{n;s/name *= *"[^"]*"/name="dspy"/;}' pyproject.toml
       - name: Update package name in metadata.py
```

Comment:
What is this step for?

Reply:
DSPy has two packages with the same content: "dspy" and "dspy-ai". This workflow replaces the name of the package dynamically.